docs/src/config/ini-config.adoc

```diff
@@ -423,33 +424,43 @@ must write RS274NGC code to standard output. This output is what will
 be displayed in the text area, previewed in the display area, and
 executed by LinuxCNC when Run.
 
-* 'PROGRAM_EXTENSION = .extension Description'
+----
+PROGRAM_EXTENSION = .extension Description
+----
 
 If your post processor outputs files in all caps you might want to add
 the following line:
 
-* 'PROGRAM_EXTENSION = .NGC XYZ Post Processor'
+----
+PROGRAM_EXTENSION = .NGC XYZ Post Processor
+----
```

Comment:
same here...

Reply:
This is an example... Which I also suspect to be wrong, missing a '#' before description